drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -18,6 +18,7 @@ class HmisSchema::CeReferral < Types::BaseObject
     field :created_at, GraphQL::Types::ISO8601DateTime, null: false
     field :current_step_name, String, null: true
     field :target_enrollment, Types::HmisSchema::Enrollment, null: true # Don't resolve in batch
+    field :swimlanes, [HmisSchema::CeReferralSwimlane], null: true
```

Comment:
we should be able to say `null: false` here, unless there is a meaningful difference between resolving an empty array and null?